rifle.py

```diff
@@ -36,17 +40,49 @@ def __init__(self, universe, DRAWING_SCALE):
         self.word_length_min_ = 3
         self.word_length_range = 3
 
-
+    def getTargetID(self,input): 
+        for enemy in self.universe_.enemies(): 
+            target_tag_word = self.target_tags_[enemy.ID_]
+            if(input == target_tag_word):
+                target_ID = self.ids_for_target_tags_[input]
+                break
+            else:       # input does not exist 
+                target_ID = -1
```

Comment:
In Python, `None` is an explicit object, you should use `None` instead of `False` or `-1` in cases like these. In C++, this would be completely acceptable though :)